### DISCUSSION TITLE
Financial support

### DISCUSSION THREAD

**Discussion by klopstock-dviz** [AntiMicroX/antimicrox#212] [Q&A] [answered]
Thanks for this soft that helped me to make the transition to linux
I want to contribute with a little donation, how to do that ?

**Comment by pktiuk** [comment, marked as answer]
Hi @klopstock-dviz,  
Thanks for your words of support, we appreciate your willingness to contribute, but we (as a project) don't need any money. We don't have any costs linked with maintenance/machines etc.  
We are developing this app during our free time as a part of our hobby, and we don't need any other gratification than knowledge that our job is useful for someone. :D 

If you really want to donate you can donate this money to any open-source foundations like [Mozilla Foundation](foundation.mozilla.org/), [Gnome Foundation](https://www.gnome.org/foundation/) or [Free Software Foundation](https://www.fsf.org/)

**Comment by klopstock-dviz** [comment]
Thank you for your replyI wish you succes




    Le mardi 1 décembre 2020 à 23:15:26 UTC+1, Paweł Kotiuk <notifications@github.com> a écrit :  
 
 


Hi @klopstock-dviz,
Thanks for your words of support, we appreciate your willingness to contribute, but we (as a project) don't need any money. We don't have any costs linked with maintenance/machines etc.
We are developing this app during our free time as a part of our hobby, and we don't need any other gratification than knowledge that our job is useful for someone. :D

If you really want to donate you can donate this money to any open-source foundations like Mozilla Foundation, Gnome Foundation or Free Software Foundation
